### DISCUSSION TITLE
Mysql connection not close after idle timeout set

### DISCUSSION THREAD

**Discussion by sagar-tts** [serverless/serverless#9819] [Q&A]
Here is code example of serverless and mysql2 with knexjs lib
in the AWS lambda function not close connection after idle time set 1000ms

used libs
- serverless
- Mysql2
- Knexjs v0.95

```
import knex from "knex";
import mysql2 from "mysql2";

if (typeof rdsClient === "undefined") {
  // Connect to the MySQL database
  var rdsClient = knex({
    client: "mysql2",
    connection: {
      host: "localhost",
      port: 3306,
      user: "test-user",
      password: "test-password",
      database: "test-db",
      pool: {
        min: 0,
        max: 1,
        idleTimeoutMillis: 1000,
      }
    }
  });

}
```

It continuously increase and DB connection and it close around after 10 minuets

Thanks,

**Comment by pgrzesik** [comment]
Hello @sagar-tts - it does not look like a bug with Serverless Framework. I'm going to move this to Discussions Q&A - maybe someone will be able to help you debug your function's configuration
